webpack.config.js

```diff
@@ -0,0 +1,155 @@
+const webpack = require('webpack');
+const TerserPlugin = require('terser-webpack-plugin');
+const path = require('path');
+const argv = require('yargs').argv;
+const pkg = require('./package');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+const IS_PRODUCTION = Boolean(argv.production);
+const VERSION = argv.production ? pkg.version : 'dev';
+const SOURCE_DIR = path.join(__dirname, 'src');
+const DIST_DIR = path.join(__dirname, 'dist');
+const VERSIONED_DIST_DIR = path.join(DIST_DIR, VERSION);
+const SOURCE_DOCS_DIR = path.join(SOURCE_DIR, 'docs');
+const SOURCE_COMPONENTS_DIR = path.join(SOURCE_DIR, 'components');
+
+const babelEnv = params => [
```

Comment:
I'd just inline it

Reply:
i use it in 2 places here and possibly will use in more places when we will introduce proper builds

Reply:
why not babelrc?